index.html

```diff
@@ -0,0 +1,1209 @@
+                </div>
+              </section>
+
+              <section id="authorization-evaluation" inlist="" rel="schema:hasPart" resource="#authorization-evaluation">
+                <h3 property="schema:name">Authorization Evaluation</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>The evaluation of an authorization is concerned with finding Authorizations that match the required parameters of an operation (<cite><a href="#authorization-conformance" rel="rdfs:seeAlso">Authorization Conformance</a></cite>). Evaluation stops when all access permission requests have been granted by one or more Authorizations. Authorizations that do not match a required access permission have no effect on the outcome of the evaluation. Access is granted when conforming Authorizations are matched, otherwise access is denied.</p>
+
+                  <p>As per determining the <cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite> of a resource, an Authorization can be evaluated explicitly (via <code>acl:accessTo</code>) or inherited (via <code>acl:default</code>) against a particular request.</p>
+
+                  <p>When a request requires an access mode (<code>acl:mode</code>) which is a limitation of another access mode, then access is granted if either mode is allowed by an Authorization. For example, when a request requires <code>acl:Append</code>, then access will be granted to agents having <code>acl:Write</code>.</p>
+
+                  <p>The presence of the <code>acl:origin</code> property and its value is taken into account in the evaluation only when the HTTP request includes the <code>Origin</code> header.</p>
```

Comment:
This feels like it needs a link to a paragraph

Reply:
Do you mean it should refer to something or that the paragraph should include the `id` attribute? If latter, I do plan to add ids to all significant statements (as well as for requirements) once this PR gets merged - no particular reason other than to have the structure/information settle down.

Reply:
I meant it needs some more details and explanation

Reply:
Ah right, '5.3.2 Web Application Authorization' would be the section to link to from here.

Reply:
https://github.com/solid/web-access-control-spec/pull/83/commits/31fee9eae7c3e5b878df8a8eba109b4bf235ffb5